bin/start_lke.sh

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env bash
+
+set -o errexit  # abort on nonzero exit status
+set -o nounset  # abort on unbound variable
+set -o pipefail # don't hide errors within pipes
+
+# Don't pollute console output with upgrade notifications
+export PULUMI_SKIP_UPDATE_CHECK=true
+# Run Pulumi non-interactively
+export PULUMI_SKIP_CONFIRMATIONS=true
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+if ! command -v pulumi >/dev/null; then
+  if [ -x "${script_dir}/../pulumi/python/venv/bin/pulumi" ]; then
```

Comment:
I just caught this and I known we do it everywhere. The `venv` directory may not be at the `../pulumi/python/venv` if someone is using pyenv. It would be best to just error here rather than potentially add the wrong path to the path. You could check for the presence of the `VIRTUAL_ENV` variable to see if a proper virtual env was loaded and warn on it.

Reply:
@dekobon - I'm inclined to leave this as-is in order to keep it inline with the rest of the code while you work on the refactor of the startup process. Thoughts?

Reply:
Yeah, leave it out.